data/themes/default/style.css

```diff
@@ -42,7 +41,7 @@ QMdiArea {
 
 lmms--gui--FileBrowser QCheckBox
 {
-	font-size: 10px;
+	font-size: 8pt;
```

Comment:
I'm assuming you did this since 10px is ~8pt? We could be accurate and say ``7.5pt``. Does this not work well with decimal based values?

Reply:
I think I chose the values so that the check boxes "User content" and "Factory content" looked good, i.e. that they are readable but not too big that they get cut off. See here:

![2510-QTreeViewFixes-FileBrowser](https://github.com/LMMS/lmms/assets/9293269/612c81fa-9918-4d99-8482-0d4b7a4f18f7)

Reply:
> I'm assuming you did this since 10px is ~8pt? We could be accurate and say `7.5pt`. Does this not work well with decimal based values?

Addendum: One cannot even convert between pixels and points anyway because how many pixels make a point should depend on the DPI of the display.

Reply:
> Addendum: One cannot even convert between pixels and points anyway because how many pixels make a point should depend on the DPI of the display.

Ah, makes sense. Sorry, just did a quick search on what the conversion would be, didn't know it depended on another variable. So is this why we should be points instead of pixels? Since they always consist of whole pixels?

Reply:
The reason that points work better with HiDPI displays is that the font rendering libraries must take the DPI values into account when rendering fonts that are defined in points. So they make sure that text will look visually the same on display devices with different DPI values.

With fonts that are defined in pixels the font library just render them such that the pixel size is reached. Because HiDPI devices have more pixels per inch this results in text looking smaller than on a display that has less pixels per inch.

Put differently: pixel size is a very technical specification while point size is more of a logical one.